pkg/data/components/OTelGRPCExporter.yaml

```diff
@@ -58,6 +65,8 @@ templates:
     data:
       - key: "{{ .ComponentName }}.endpoint"
         value: "{{ firstNonZero .HProps.Host .User.Host .Props.Host.Default }}:{{ firstNonZero .HProps.Port .User.Port .Props.Port.Default }}"
+      - key: "{{ .ComponentName }}.tls.insecure"
+        value: {{ firstNonZero .HProps.Insecure .User.Insecure .Props.Insecure.Default }}
```

Comment:
Suggestion: use suppress_if to only include this if the result is true.

Reply:
good call, used suppress and added a test for insecure